alfasim_sdk/cli.py

```diff
@@ -70,8 +75,11 @@ def _compile(plugin_dir):
         raise FileNotFoundError(
             f"Was not possible to find a compile.py file in {plugin_dir}"
         )
+    import alfasim_sdk
 
-    subprocess.run(["python", str(compile_script)])  # nosec
+    _env = os.environ.copy()
```

Comment:
Why `_env` and not simply `env`?

Reply:
Well, I just  don't have any good and valid arguments to justify this atrocity 😝